README.md

```diff
@@ -54,13 +54,13 @@ Or individual tests
 ### Development Server
 
 A cloned repository is deployed to the development server and run in development mode.
-This is not the usual way Capistrano is used, but is done so to enable development work directly on the
+This is not the usual way Capistrano is used, but is done to enable development to work directly on the
```

Comment:
This change does not read the way we intended. We are saying that we want Capistrano to do it the special way so that we are enabled to do development work directly on the server.
```
This is not the usual way Capistrano is used, but is done so to enable development work directly on the server.
```
If we were to use the normal way - loading production - we would not be able to do development work because our development group gems would not be loaded.